release-notes/EPM/I-DOCSIS/EPM_7.0.8_Integrated_DOCSIS.md

```diff
@@ -0,0 +1,41 @@
+---
+uid: EPM_7.0.8_Integrated_DOCSIS
+---
+
+# EPM 7.0.8 Integrated DOCSIS - Preview
+
+> [!IMPORTANT]
+> We are still working on this release. Some release notes may still be added, modified, or moved to a later release. Check back soon for updates!
+
+## Changes
+
+### Enhancements
+
+#### CISCO CBR-8 CCAP Platform: CCAP TV data added [ID 41139]
+
+Two new tables are now available in the CISCO CBR-8 element: the *Video QAM Channel* table and the *MPEG Output Streams* table. These tables are shown on the *Video Data* page of the element. You can also retrieve the data from these tables in a dashboard or low-code app. If you do not need this TV data for your implementation, leave the *Poll Video Data Tables* parameter set to its default setting *Disabled*, so that this data will not be polled.
```

Comment:
Can you include the OIDs of the SNMP tables?  It is not required to describe all the columns, only the table SNMP OID.
Instead of *You can also retrieve the data from these tables in a dashboard or low-code app* it should be *you can also visualize data from these tables in a dashboard or low-code app".
Instead of *If you do not need this TV data ....* it should be *By default these tables are not polled. Polling this table can be controlled using the parameter Poll Video Data Tables*. Please indicate where this parameter is located.
Can we also update the title of this feature? It should be *Cisco CBR-8 CCAP Platform: CCAP video data added*.

Reply:
@SkylineGBO, I've implemented most of this feedback, but I don't know what the OIDs of the SNMP tables are. Could you provide this information?

Reply:
apologies for the late reply, here are the oids:
•	Video QAM Channel- 1.3.6.1.4.1.5591.1.11.5.3.1.1.1
•	MPEG Output streams - 1.3.6.1.4.1.5591.1.11.5.4.1.1.2.3